robot/basestation/templates/elements/feed/ArmMotorTable.html

```diff
@@ -12,39 +12,39 @@ <h5 class="container-header">Motor Table</h5>
     <tr>
       <th scope="row">M1 (Shoulder Swivel)</th>
       <!-- @TODO: add backend code to update status/motor current-->
-      <td id="m1-status">N/A</td>
-      <td id="m1-angle">-</td>
-      <td id="m1-current">N/A</td>
+      <td class="table-text" id="m1-status">N/A</td>
```

Comment:
Since CSS inherently supports inheritance, it may save us some repetition if you declare the class `table-text` in the `<tr>` element. Or are you in fact trying to only apply it to those `td`s only?

Reply:
Only the `td`s

Reply:
Fair enough